bounties/npm/@fav/prop.assign-deep/1/vulnerability.json

```diff
@@ -0,0 +1,56 @@
+{
+  "PackageVulnerabilityID": "2",
```

Comment:
Spotted the culprit - change this to `1`.

Reply:
Corrected!
Thanks @JamieSlome

Reply:
@JamieSlome Still the tests are failing.

Reply:
Yep, that fixed one issue. Just fixing a bug on our side for the CVSS checks.